run-ib-igprof

```diff
@@ -4,11 +4,16 @@ case $CMS_BOT_DIR in /*) ;; *) CMS_BOT_DIR=$(pwd)/${CMS_BOT_DIR} ;; esac
 WORKFLOWS=$1
 PROFILES=$2
 EVENTS=$3
+HPUWF="23434.21"
 if [ "X$EVENTS" = "X" ] ; then EVENTS=1000; fi
 for prof in ${PROFILES} ; do
   mkdir -p $WORKSPACE/igprof/${prof}
   cd $WORKSPACE/igprof/${prof}
   runTheMatrix.py $WORKFLOWS --ibeos --command "-n $EVENTS --profile $prof" 2>&1 | tee runTheMatrix.log
+  if [[ $WORKFLOWS == *"$HPUWF" ]]; then
+    edmEventSize -v step3*.root > step3_sizes_${HPUWF}.txt
```

Comment:
is not `runTheMatrix.py` create a subdirectory for every workflow and root files are under that sub-directory? So how these commands are going to find these step3*.root files?

if would suggest to do something like
```
for hpfw in ${HPUWFS} ; do
  for s in step3 step4 ; do
    if [ $(ls -d ${hpwf}_*/${s}_*.root | wc -l) -eq 0 ] ; then continue ; fi 
    edmEventSize -v ${hpwf}_*/${s}_*.root> step4_sizes_${HPUWF}.txt
done
```

Reply:
The root files are produced when igprof is run and they should be in the same directory.

Reply:
You are correct. I will update this PR.